docs/loadjdbc.adoc

```diff
@@ -457,4 +457,53 @@ You can call the procedure without param:
 [source,cypher]
 ----
 CALL apoc.load.jdbcUpdate('jdbc:mysql:....','INSERT INTO RECOMMENDATIONS values(user.id, reco.id, score)') YIELD row;
+----
+
+
+=== Load JDBC format date
+
+[source, cypher]
+----
+CALL apoc.load.jdbc('key or url','table or kernelTransaction', config) YIELD row
+----
+
+=== Config
+
+Config param is optional, the default value is an empty map.
+
+[cols="3m,2"]
+|===
+|localDateTime| value: true/false i.e "2007-12-03T10:15:30"
+|localDate| value: true/false i.e "2007-12-03"
+|zoneDateTime| value: true/false i.e "2007-12-03T10:15:30+01:00 Europe/Paris"
+|offSetTime| value: true/false i.e "2007-12-03T10:15:30+01:00"
+|timezone| default value: 'UTC'
```

Comment:
Can we have an example for timezone and the different types?

Reply:
Done

Reply:
Done